pkg/runtime/runtime.go

```diff
@@ -1135,15 +1168,15 @@ func (a *DaprRuntime) publishMessageHTTP(ctx context.Context, msg *pubsub.NewMes
 
 	var span *trace.Span
 
-	route := a.topicRoutes[msg.Metadata[pubsubName]].routes[msg.Topic]
+	route := a.topicRoutes[msg.metadata[pubsubName]].routes[msg.topic]
```

Comment:
@artursouza qq how and why did the var name change from `Metadata` to `metadata` and from `Topic` to `topic`?

Reply:
It is a new internal structure and not the one from contrib anymore.